dotnet/src/support/PageObjects/ByAll.cs

```diff
@@ -0,0 +1,79 @@
+﻿/*
+Copyright 2007-2015 Selenium committers
```

Comment:
@ddavison said I should put in that exact text, but whatever suits you best.
So it should be "Copyright 2015 Software Freedom Convervancy", right?
(oh the woes of putting a copyright notice in every file ;))